docker-compose-local.yml

```diff
@@ -75,7 +75,7 @@ services:
       - AWS_EXECUTION_ENV=True
       - DEFAULT_REGION=eu-west-2
     volumes:
-      - "${TMPDIR:-/tmp/localstack}:/tmp/localstack"
+      - "${LOCALSTACK_VOLUME_DIR:-./volume}:/var/lib/localstack"
```

Comment:
is it ok to push this in this PR?

Reply:
Well you know the answer to that,.. it's not part of PR but I thought I put it in